recipes/upterm/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euxo pipefail
+
+export LDFLAGS="-s -w -X github.com/owenthereal/upterm/internal/version.Version=${PKG_VERSION}"
+
+go build -ldflags "${LDFLAGS}" -o "${PREFIX}/bin/upterm" ./cmd/upterm
+go build -ldflags "${LDFLAGS}" -o "${PREFIX}/bin/uptermd" ./cmd/uptermd
```

Comment:
```suggestion
export LDFLAGS="${LDFLAGS} -X github.com/owenthereal/upterm/internal/version.Version=${PKG_VERSION}"

go build -ldflags "${LDFLAGS}" -o "${PREFIX}/bin/upterm" ./cmd/upterm
go build -ldflags "${LDFLAGS}" -o "${PREFIX}/bin/uptermd" ./cmd/uptermd
```

you should keep the conda-forge wide LDFLAGS